flipperdevices/flipperzero-firmware
# CCID: Improve request and response data handling

# What's new

 - Add iso7816_set_response function: serves a helpers to set SW1 and SW2 values
 - improved iso7816_read_response_apdu by correctly parsing Lc and Le values
 - add client script to make testing easier

# Verification 

- Start CCID Test application
- run the ccid_client.py script under applications/debug/ccid_test/client
- Select the 'Generic USB Smart Card Reader'
- All tests should pass

# Checklist (For Reviewer)
